Rafarinh0/sociologia-bot
# Feature/oxe command

# What I did

- Added oxe command
- Added Pull Request template

## How to test

1. Open the terminal and go to the folder of this project

1. In terminal:
   1. `git checkout master`
   1. `git pull`
   1. `git checkout feature/oxe command`
   1. `git pull origin feature/oxe command`
   1. `yarn install`
   1. `yarn start`

1. Enter a voice channel, type !oxe on a text channel and see if Kern says Ooooxe, and the bot replies Ooxe to you.

1. Read the changed files section on GitHub

1. Test the things that you think are worth testing, even the ones that are not in this description
